src/docker/DockerServeClient/DockerServeUtils.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Container } from '@docker/sdk/containers';
+import { DockerContainer, InspectionPort } from '../Containers';
+
+export function containerToDockerContainer(container: Container.AsObject): DockerContainer | undefined {
+    if (!container) {
+        return undefined;
+    }
+
+    const ports = container.portsList.map(p => {
+        return {
+            IP: p.hostIp,
+            PublicPort: p.hostPort,
+            PrivatePort: p.containerPort,
+            Type: p.protocol,
+        };
+    });
+
+    const labels: { [key: string]: string } = {};
+    container.labelsList.forEach(l => {
+        const [label, value] = l.split('=');
+        labels[label] = value;
+    });
+
+    return {
+        Id: container.id,
+        Image: container.image,
+        Name: container.id, // TODO ?
+        State: container.status,
+        Status: container.status,
+        ImageID: undefined, // TODO ?
+        CreatedTime: undefined, // TODO ?
+        Labels: labels, // TODO--not working

```

Comment:
Presumably it will work once the SDK supports local Docker engine. It does not work with ACI because ACI does not support labels.

What I am saying let's put a better comment here ☺️